### ISSUE TITLE
filter bar in posts widget please

### ISSUE THREAD

**Issue by dasigna** [pojome/elementor#5916]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
i'm very frustrated when trying to show a bunch of posts with the post grid widget and not being able to sort them via filter bar on frontend!

sorry for bringing this up again, but as a long term request this seems to be abandoned???

again facing the situation where the portfolio widget is no useable substitute at all... it is great to have call for custom fields and custom queries now (absolutely love it!) but the linking between all those possibilities is still quite frustratin at the time being (sorry to say this).

so i am able to have future posts as events - great!
show them as a post grid - great!
trying to implement custon fields (date, location, cost, type ...) very hard to achieve and currently only possible with a workaround using the great eleskin-plugin from @dudaster (thanks for this, mate) but naturally with lots of limitations.
and if i am now trying to sort a bunch of events on the frontend e.g. type or location of event according to its category i am completely stuck!!
no chance to use the portfolio instead because of the missing chance to show more info (or simply dates - as there is only the title ...)

so use another plugin?? please - not for this simple scenario ...

**Describe the solution you'd like**
simply put the filter bar option into the posts widget - done!
and if you allow calling custom fields through post widget it just would be close to perfect :)
allow custom skins too and you are king!!!

**Describe alternatives you've considered**
there are none - currently. apart from using another page-builder.

**Additional context**
just filter bar for posts widget at the moment and the first deed is done. big thanks in advance.

**Comment by dasigna** [comment]
... just forgot to mention:
the post navigation widget is just quite useless too when trying to use it with a custom query for cpt.
if the post navigation cant follow the content of cpts with custom query its quite annoying - so custom query for post navigation widget too, please.

**Comment by stebla27** [comment]
+1

**Comment by black-eye** [comment]
+1

**Comment by bainternet** [comment]
Duplicate of #3140

**Comment by bainternet** [comment]
please avoid opening duplicate issues knowingly.

**Comment by dasigna** [comment]
serious?
sorry for that, but we wouldnt be tempted to do so if there would be some clear reaction from you, besides from just closing those issues/requests repatingly :-)

**Comment by black-eye** [comment]
Agreed with @dasigna - the lack of communication from devs is really frustrating. That's why the temptation for opening new issues again and again.

Not to mention, that #3140 is just a uplicate of #1716 which was shut right in the beginning and even though gained 22 supporters.

So please explain us the open/close policy for the issues and the way how to gain some reaction on the issue. Not to mention some roadmap. Now it seems that the policy is the noble "silence is golden".

Don't get me wrong - I don't want to be rude or harsh here and I really like most of the Elementor's features. But the way how you handle the feature requests and suggestions from supporting community is, well ... not very supporting.

**Comment by dasigna** [comment]
***sigh***

also no offend from my side at all!
but in fact its kind of frustrating when threads are closed immediately or lacking of more precise feedback.

elementor is great, no doubt. and so are the latest features. so dont get me wrong at all please.
but completely agreeing to @black-eye and others here in terms of reaction from elementor and lack of roadmap etc.

i know its getting harder every time the community grows and its getting harder to maintain this repo as the number of involved people grows and throwing in request after request often without reading first and elaborating whats already suggested - and so on.
but again at this point a clearer communication and readable roadmap would help a lot.
to me it seems there is hard work on both sides, so lets act with respect and open mind.

thanks.

EDIT:
just to mention it as an aside - when contacting support after being asked to do so, the first respond almost always lacks of precision and clear referring to the issue too.... sadly. you'll almost every time need two rounds or more till the inital complaint is being read well and understood too (and thats mostly not a problem of not describing the issue precise enough from the client side...)
further on: once you've given credentials to a site to study the described problem live, you've been asked first to submit a screencast. but this will take me longer than it will for any dev to take a quick look himself. thats completely frustrating again.

**Comment by bainternet** [comment]
@dasigna @black-eye 


I'll try to explain our open/close policy for the issues here:

* Bug report:
    - Once the bug is confirmed its labeled as "BUG"
    - issues are closed once they are fixed.
* Feature requests: 
    - As long as the issue is opened and labeled with "Feature request" it will get addressed, either via a feature implementation, or an alternative solution suggestion.
    - If its closed otherwise, then the reason for closing will always be xplained  in the issue prior to closing.

@dasigna 
Thanks for the feedback about the support, I'll pass it on, but I do understand the requests for screencast requests which are the best way to understand the issue in hand.

thanks again.

**Comment by black-eye** [comment]
@bainternet OK, taking #3140 as an example - opened, but with no label "Feature request". What's the status here then? Opened on jan 11, no comments from devs yet. 

Original request #1716 closed with no further explanation and stayed closed even after ongiong requets - thus somebody opened new issue again. That is what we are talking about...

**Comment by dasigna** [comment]
agree with @black-eye, jep.
excactly thats the temptation to open a new one ... one doesnt really know whats going on.

@bainternet - thanks for explanation. and: i agree regarding the screencasts. but it would be much faster and more efficient to use given credentials and take a quick look if the issue is described quite well - which was the case. and it would provide the best opportunity to do a effective research immediately  :)

**Comment by hbodell** [comment]
+1

**Comment by Jordidriesen** [comment]
+1

**Comment by bcslaam** [comment]
+1 The posts grid is a major part of many sites. We need a filter bar....please.
What is the developer thinking? That people wont jump ship and they can just sit on their laurels? Dont think so guys. Wake up!

**Comment by davidamarek** [comment]
+1

**Comment by nicholaszein** [comment]
Heli

Thank you for your submission! 🙏 

### 🔗 This issue is a Duplicate of #3140

Which is on the same subject. 

As @bainternet said, this issue has been 🔃 **Merged** with the original, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

Best regards,

Nick
